05/app.js

```diff
@@ -1,53 +1,59 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+	paragraphs: {
+		p1: 0,
+	},
+	links: {
+		'/dolor.html': 0,
+	},
 };
 
 /* tutaj umieść swój kod */
-
+const pElement = document.querySelector('[data-id="p1"]');
+pElement.addEventListener('click', function (e) {
+	if (e.target.className === 'text') {
+		stats.paragraphs.p1++;
```

Comment:
Kierunek dobry, ale zauważ, że tam są paragrafy o różnym `id` - należałoby pobrać to `id` i wykorzystać tj. `stats.paragraphs[id]++`. Dodatkowo musisz zwrócić uwagę, że kolejne tj. `p2` oraz `p3` jeszcze nie zostały zadeklarowane więc trzeba to zrobić przed dodaniem czegokolwiek do nich :)